tests/build/build.go

```diff
@@ -123,6 +128,241 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 
 		})
 
+		When("a new simple build component is created", Label("annotations"), func() {
+			var component *appservice.Component
+
+			BeforeAll(func() {
+				componentObj := appservice.ComponentSpec{
+					ComponentName: annotationsComponentName,
+					Application:   applicationName,
+					Source: appservice.ComponentSource{
+						ComponentSourceUnion: appservice.ComponentSourceUnion{
+							GitSource: &appservice.GitSource{
+								URL:      helloWorldComponentGitSourceURL,
+								Revision: "",
+							},
+						},
+					},
+				}
+				component, err = f.AsKubeAdmin.HasController.CreateComponent(componentObj, testNamespace, "", "", applicationName, false, nil)
+				Expect(component).ToNot(BeNil())
```

Comment:
Will it be good to add a message in case this fails like `got unexpected response nil while creating component`